### DISCUSSION TITLE
webrtc multiple input to one output stream

### DISCUSSION THREAD

**Discussion by falcohulzebos** [OvenMediaLabs/OvenMediaEngine#1290] [Help]
I am trying to setup a scenario where 2 or more people can stream to the server over webrtc (whip) and then the inputs get combined into one output stream. Is this possible?

**Comment by bchah** [comment]
Are you looking for functionality like [StreamYard](streamyard.com) or [Melon](melonapp.com)? I don't think OME is intended to act as a "vision mixer" in that same way. You'd probably do best to use one of those tools or something like [Wirecast](https://www.telestream.net/wirecast/overview.htm) to gather your inputs, arrange them *just perfect* and send the resulting output to OME for distribution.
